Move External Identities APIs to v1.0 (#12210)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/clientcertificateauthentication.md

---
title: "clientCertificateAuthentication resource type"

> description: "Represents configuration for fetching a clientCertificateAuthentication in an API call."

Reviewer: ```suggestion
description: "Represents the configuration for retrieving a clientCertificateAuthentication."
```

author: "nickgmicrosoft"
localization_priority: Normal
ms.prod: "identity-and-sign-in"
doc_type: resourcePageType
---

# clientCertificateAuthentication resource type

Namespace: microsoft.graph

> A type derived from apiAuthenticationConfigurationBase which is used to represent Pkcs12 based client certificate authentication. This is used to retrieve the public properties of uploaded certificates.

Reviewer: ```suggestion
A type derived from apiAuthenticationConfigurationBase that is used to represent a Pkcs12-based client certificate authentication. This is used to retrieve the public properties of uploaded certificates.
```
